### PR TITLE
feat: final quiz for NFT-Front-end lesson 5

### PR DESCRIPTION
**Pull request by elPiablo**
## Changes made:

- [x] add first 5 of ~12-15 questions
- [x] add next 5 questions
- [x] add remaining final quiz questions

## Notes to reviewers:
- Ready for review. The more that is on the `main` branch, the better for everyone, partner authors included. I will most likely need to come back and make a few adjustments after Brian has made enhancements to the lesson itself.

### CODE REVIEW

File: apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        },
+        {
+          "answer": "A styling tool for stacking designs on the user interface"
+        }
+      ]
+    },
+    {
+      "question": "What did we need to get from the the WalletConnectCloud?",
+      "options": [
+        {
```
Comment (elPiablo) [inline apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json:79]: Q5. it is possible answer a) is also correct

File: apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+          "answer": "A connection to cloud storage for the NFTs"
+        }
+      ]
+    },
+    {
+      "question": "Why do we create a custom `mintToken` function?",
+      "options": [
+        {
+          "answer": "To handle error messages"
```
Comment (elPiablo) [inline apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json:95]: Q6. it is also possible that answer a) in this one is also correct

Comment (elPiablo) [reply apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json:95]: I'm going to leave the first answer a) out in the next commit = there will only be three possible answers

**Review by Markkos89 — APPROVED** [review]
:raised_hands:

File: apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+        {
+          "answer": "A programming language"
+        },
+        {
+          "answer": "A styling tool for stacking designs on the user interface"
+        }
+      ]
+    },
+    {
+      "question": "What did we need to get from the the WalletConnectCloud?",
```
Comment (briangershon) [inline apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json:74]: Duplicate of "the"

File: apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+          "answer": "To manage changes and side effects from its own functions in the hook",
+          "correct": true
+        }
+      ]
+    },
+    {
+      "question": "We talk about the `TierABI` in the lesson. What is it?",
+      "options": [
+        {
+          "answer": "A React library that let’s us mint our NFT in tiers"
```
Comment (briangershon) [inline apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json:59]: I think it should be "lets us"

File: apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+          "answer": "The variable is defined with the `public` keyword",
+          "correct": true
+        }
+      ]
+    },
+    {
+      "question": "Which of the following shows on our modal footer?",
+      "options": [
+        {
+          "answer": "Join Developer_DAO"
```
Comment (briangershon) [inline apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json:251]: Is the underscore an official part of the Developer DAO name?

(I removed it in my Lesson 5 PR... lemme know if I should add it back)

**Review by briangershon — APPROVED** [review]
Thanks for all the quiz questions!

I just had a couple of comments.

ps: I wasn't able to run and try this locally for some reason. I could see Quiz question 1, but clicking on it didn't work.

File: apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json (hunk 4 above)
Comment (elPiablo) [reply apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json:59]: haha, you've seen a lot of those 'let's' recently 🤣

**Comment by elPiablo** [comment]
> ps: I wasn't able to run and try this locally for some reason. I could see Quiz question 1, but clicking on it didn't work.

Hey Brian, 
The v2 version of the quiz isn't fully functioning. you should be able to connect your wallet, and should be able to see all the questions and possible answers from A to Z locally. As you mentioned, clicking on the answer works, but doesn't select them further than that. Still needs to be fixed. 

Thanks for taking the time to do the review ✊🏼

**Comment by elPiablo** [comment]
> 🙌

Thanks @Markkos89 holiday maker!! 🤣

**Pull request merged**